### PR TITLE
Creating a schema index page

### PR DESCRIPTION
**Pull request by Guzel738**
Schema index page created, it is the part of the schema_registry_views feature branch.
Tests will be later.

### CODE REVIEW

**Review by workshur — APPROVED** [review]
👍

File: kafka-ui-react-app/src/redux/actions/actions.ts [hunk 1]
```diff
@@ -97,3 +98,9 @@ export const fetchConsumerGroupDetailsAction = createAsyncAction(
   { consumerGroupID: ConsumerGroupID; details: ConsumerGroupDetails },
   undefined
 >();
+
+export const fetchSchemasByClusterNameAction = createAsyncAction(
+  ActionType.GET_CLUSTER_SCHEMAS__REQUEST,
```
Comment (aMoRRoMa) [inline kafka-ui-react-app/src/redux/actions/actions.ts:103]: @workshur Does this not conflict with your updates?

File: kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { SchemaSubject } from 'generated-sources';
+import ActionType from 'redux/actionType';
+import { Action, SchemasState } from 'redux/interfaces';
+
+export const initialState: SchemasState = {
+  byName: {},
+  allNames: [],
+};
+
+const updateSchemaList = (
```
Comment (aMoRRoMa) [inline kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts:10]: I would suggest doing all manipulation with data in thunks and in reducer just apply it.

**Review by aMoRRoMa — APPROVED** [review]
Good work!

File: kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        },
+      },
+      allNames: [...memo.allNames, schema.subject],
+    };
+  }, initialMemo);
+};
+
+const reducer = (state = initialState, action: Action): SchemasState => {
+  switch (action.type) {
+    case ActionType.GET_CLUSTER_SCHEMAS__SUCCESS:
```
Comment (soffest) [inline kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts:37]: This one will also conflict with @workshur's #186

**Review by soffest — APPROVED** [review]

File: kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts (hunk 3 above)
Comment (workshur) [reply kafka-ui-react-app/src/redux/reducers/schemas/reducer.ts:37]: It is expected

**Pull request merged**